nixio/block.py

```diff
@@ -362,8 +362,12 @@ def find_sources(self, filtr=lambda _: True, limit=None):
             limit = maxint
         return finders._find_sources(self, filtr, limit)
 
-    def pprint(self, indent=2, max_length=120, extra=True):
-        print(self)
+    def pprint(self, indent=2, max_length=120, extra=True, file=False):
+        if file:
```

Comment:
If the `file` argument only controls indent, and the indent can be set from the calling function, why can't not increment the indent from the `file.pprint()` caller?